src/views/Pipeline.vue

```diff
@@ -554,11 +555,23 @@ export default defineComponent({
 
       await this.store.dispatch('user/updatePinnedJobs', { pinnedJobs: [...pinnedJobs] });
       this.updateSelectedPinnedJob(enumId)
+    },
+    updateCurrentJob() {
+      if(!this.getCurrentJob) {
```

Comment:
This should be handled properly. I think we should check if it is desktop or not

Reply:
Sir, Updated code.